### PR TITLE
Add CiliumAPITooSlow

### PR DESCRIPTION
**Pull request by kopiczko**
Towards: https://github.com/giantswarm/giantswarm/issues/30878

### CODE REVIEW

File: helm/prometheus-rules/templates/platform/cabbage/alerting-rules/cilium.rules.yml [hunk 1]
```diff
@@ -33,6 +33,18 @@ spec:
...
         team: cabbage
         topic: cilium
+    - alert: CiliumAPITooSlow
+      annotations:
+        description: '{{`Cilium API processing time is >50s pod="{{ $labels.pod }}" node="{{ $labels.node }}" method="{{ $labels.method}}" path="{{ $labels.path }}"`}}'
+        opsrecipe: cilium-performance-issues/
+      expr: avg(rate(cilium_agent_api_process_time_seconds_sum{}[5m])/rate(cilium_agent_api_process_time_seconds_count{}[5m]) > 50) by (cluster_id, pod, method, path)
+      for: 20m
+      labels:
+        area: managedservices
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/platform/cabbage/alerting-rules/cilium.rules.yml:43]:
```suggestion
        area: platform
```

or is it kaas?

File: helm/prometheus-rules/templates/platform/cabbage/alerting-rules/cilium.rules.yml [hunk 2]
```diff
@@ -33,6 +33,18 @@ spec:
         severity: page
         team: cabbage
         topic: cilium
+    - alert: CiliumAPITooSlow
+      annotations:
+        description: '{{`Cilium API processing time is >50s pod="{{ $labels.pod }}" node="{{ $labels.node }}" method="{{ $labels.method}}" path="{{ $labels.path }}"`}}'
+        opsrecipe: cilium-performance-issues/
+      expr: avg(rate(cilium_agent_api_process_time_seconds_sum{}[5m])/rate(cilium_agent_api_process_time_seconds_count{}[5m]) > 50) by (cluster_id, pod, method, path)
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/platform/cabbage/alerting-rules/cilium.rules.yml:40]: lint says you're missing some labels here :)

**Comment by QuentinBisson** [comment]
See linting:

```
test/hack/output/generated/capi/capa-mimir/platform/cabbage/alerting-rules/cilium.rules.yml:29 Bug: Template is using `node` label but the query removes it. (alerts/template)
 29 |           description: 'Cilium API processing time is >50s pod="{{ $labels.pod }}" node="{{ $labels.node }}" method="{{ $labels.method}}" path="{{ $labels.path }}"'

test/hack/output/generated/capi/capa-mimir/platform/cabbage/alerting-rules/cilium.rules.yml:31 Bug: `installation` label is required and should be preserved when aggregating `^.+$` rules, use `by(installation, ...)`. (promql/aggregate)
 31 |         expr: avg(rate(cilium_agent_api_process_time_seconds_sum{}[5m])/rate(cilium_agent_api_process_time_seconds_count{}[5m]) > 50) by (cluster_id, pod, method, path)

test/hack/output/generated/capi/capa-mimir/platform/cabbage/alerting-rules/cilium.rules.yml:31 Bug: `pipeline` label is required and should be preserved when aggregating `^.+$` rules, use `by(pipeline, ...)`. (promql/aggregate)
 31 |         expr: avg(rate(cilium_agent_api_process_time_seconds_sum{}[5m])/rate(cilium_agent_api_process_time_seconds_count{}[5m]) > 50) by (cluster_id, pod, method, path)

test/hack/output/generated/capi/capa-mimir/platform/cabbage/alerting-rules/cilium.rules.yml:31 Bug: `provider` label is required and should be preserved when aggregating `^.+$` rules, use `by(provider, ...)`. (promql/aggregate)
 31 |         expr: avg(rate(cilium_agent_api_process_time_seconds_sum{}[5m])/rate(cilium_agent_api_process_time_seconds_count{}[5m]) > 50) by (cluster_id, pod, method, path)
```

File: helm/prometheus-rules/templates/platform/cabbage/alerting-rules/cilium.rules.yml (hunk 1 above)
Comment (kopiczko) [reply helm/prometheus-rules/templates/platform/cabbage/alerting-rules/cilium.rules.yml:43]: It's platform, I just opy/pasted from the one below

**Review by QuentinBisson — APPROVED** [review]
LGTM :)

**Pull request merged**